pftools/python/parflow/tools/builders.py

```diff
@@ -596,13 +596,22 @@ def apply(self, run=None, name_registration=True):
                 (e.g., GeomNames) related to the loaded subsurface properties
         """
         # applying subsurface properties to run keys
+        print("applying")
         if run is None:
             if self.run is None:
                 print('No run object assigned')
                 sys.exit(1)
         else:
             self.run = run
 
+        # new names are user specified names such as s1 s2 s3 for soil permeability
```

Comment:
This is the code that allows us to keep well names in the excel instead of needing to enter them all in the python file